src/test/java/uk/gov/pay/api/it/CachingAuthenticatorIT.java

```diff
@@ -101,9 +101,11 @@ private String aPayment() {
                 .add("amount", 100)
                 .add("language", "en")
                 .add("reference", "ref 12")
-                .add("state", new PaymentState("created", false, null, null))
+                .add("state", new PaymentState("created", false, null, null, null))
                 .add("email", "test@example.com")
                 .add("description", "description")
+                .add("mandate_id", "amandateid")
+                .add("provider_id", "aproviderid")
```

Comment:
This looks like it's just for card payments so don't think we should add these her.e